robot/description/launch/caffeine_gazebo.launch

```diff
@@ -1,19 +1,33 @@
 <launch>
    <arg name="use_gui" default="true"/>
    <arg name="debug_mode" default="false"/>
-   <arg name="world" default="$(find husky_gazebo)/worlds/clearpath_playpen.world"/>
-
-   <!-- Initial Pose (NOTE: angles in radians) -->
-   <arg name="x" value="0"/>
-   <arg name="y" value="0"/>
-   <arg name="z" value="0"/>
-   <arg name="roll" value="0"/>
-   <arg name="pitch" value="0"/>
-   <arg name="yaw" value="0"/>
-   
+   <arg name="world" default="walls"/>
+   <arg name="spawn" default="start"/>
+
+   <!-- IGVC worlds -->
+   <arg name="world_file" value="$(find worlds)/gazebo_worlds/igvc_walls.world"  if="$(eval arg('world') == 'walls')"/>
+   <arg name="world_file" value="$(find worlds)/gazebo_worlds/igvc_ramp.world"   if="$(eval arg('world') == 'ramp')"/>
+   <arg name="world_file" value="$(find worlds)/gazebo_worlds/igvc_plain.world"  if="$(eval arg('world') == 'plain')"/>
```

Comment:
given these if statements, maybe we want to launch the gazebo world in a separate launch file, which we call from this one?

Reply:
I agree this edit would be cleaner (will implement)